src/probes/telem_record_gen.c

```diff
@@ -48,6 +49,7 @@ static const struct option prog_opts[] = {
         { "payload", required_argument, 0, 'p' },
         { "payload-file", required_argument, 0, 'P' },
         { "record-version", required_argument, 0, 'R' },
+        { "event-id", optional_argument, 0, 'e' },
```

Comment:
Shouldn't this be a required_argument? If the user wants a static event ID, I imagine they must supply a value.

Reply:
Event id's are optional, if one is provided this will override a randomly generated during headers initialization.

Reply:
Yes, but my point was that users won't pass -e/--event-id if they want a randomly generated ID. If they *do* pass the option, then I would expect an argument to be required.